manifest.toml

```diff
@@ -37,6 +37,13 @@ ram.runtime = "50M"
     type = "group"
     default = "visitors"
 
+    [install.phpversion]
+    ask.en = "Choose a PHP version you want to use for your app"
+    ask.fr = "Choisissez une version PHP que vous souhaitez utiliser pour votre application"
+    type = "select"
+    choices = ["8.0", "8.1", "8.2"]
+    default = "8.2"
```

Comment:
Or i dunno, we should at least add an `help` key explaining that the default value is fine for regular use case and that some plugins are only compatible with older versions of php...